package.json

```diff
@@ -72,7 +72,7 @@
     "json-schema-to-ts": "^3.0.0",
     "null-loader": "^4.0.0",
     "outdent": "^0.7.1",
-    "prettier": "^2.1.2",
+    "prettier": "2.8.8",
```

Comment:
Did you mean to update the package.json file?

Reply:
I recommend you revert this change, it's possible that the updated tool version changes its behaviour and causes the prettier problems that are failing the build.